pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -655,12 +655,19 @@ in makeScopeWithSplicing' {
   # The full kernel. We do the funny thing of overridding the headers to the
   # full kernal and not vice versa to avoid infinite recursion -- the headers
   # come earlier in the bootstrap.
-  sys = self.sys-headers.override {
+  sys = self.sys-headers.override (attrs: {
     pname = "sys";
     installPhase = null;
     noCC = false;
     dontBuild = false;
-  };
+    postPatch = ''
+      ${attrs.postPatch}
+      substituteInPlace sys/conf/Makefile.kern.inc --replace "-Wa,--fatal-warnings" ""
```

Comment:
Could we do this without a patch?  Something like putting `-Wa,--no-fatal-warnings` in NIX_CFLAGS_COMPILE?

Also, it would be nicer to do these things in sys-headers, to keep them as close as possible.

Reply:
Sadly I wasn't able to override the flag.

Moved to sys-headers.